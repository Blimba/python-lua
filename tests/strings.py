print("test" + "ing")
print("test{}".format("ing2"))
str = "ab{}cb{}"
print(str.replace("b","d"))
print(str.find("c"))
print(str.format("asdf","asdf2"))
print(str[1:5])

print("()".join(str.split("{}")))

print("""a
multiline
string
""")