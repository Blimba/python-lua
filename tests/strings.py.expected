testing
testing2
ad{}cd{}
4
abasdfcbasdf2
b{}c
ab()cb()
